src/hackerspace_online/settings/base.py

```diff
@@ -394,6 +394,7 @@
         # os.path.join(STATIC_URL, 'summernote-ext-emoji-ajax/summernote-ext-emoji-ajax.css'),
         os.path.join(STATIC_URL, 'summernote-add-text-tags/summernote-add-text-tags.css'),
         os.path.join(STATIC_URL, 'summernote-list-styles/summernote-list-styles.css'),
+        os.path.join(STATIC_URL, 'css/custom_summernote_widget.css'),
```

Comment:
Will this effect the default (light) theme as well?  These styles are specific to the alternate slate (dark) theme.  Should probably name the file accordingly: `custom_summernote_widget_slate.css` or something like that to make it clear.